app/helpers/active_admin/display_helper.rb

```diff
@@ -24,7 +24,7 @@ def DISPLAY_NAME_FALLBACK.inspect
     def display_name(resource)
       unless resource.nil?
         result = render_in_context(resource, display_name_method_for(resource))
-        if result.to_s&.strip&.present?
+        if result.to_s.strip.present?
```

Comment:
unless I'm missing something and in some circumstances `to_s` can be nil (maybe override `to_s`, but that is a user problem)

Reply:
I made this update in #6342 but I don't recall why I used the safe navigation operator. Likely just out of caution without really knowing that `nil.to_s` will return an empty string so it is redundant as you say. This looks good to me and makes sense to change. Thank you.